docs/examples/federated_statistics.rst

```diff
@@ -57,7 +57,7 @@ The main steps are:
     * provide server side configuration to specify target statistics and their configurations and output location
     * implement the local statistics generator (statistics_spec)
     * provide client side configuration to specify data input location
-    * The detailed example instructions can be found in `Data frame statistics <https://github.com/NVIDIA/NVFlare/tree/main/examples/federated_statistics/df_stats/README.md>`_
+    * The detailed example instructions can be found in `Data frame statistics <https://github.com/NVIDIA/NVFlare/tree/main/examples/federated_statistics/df_stats/README.md>`__
```

Comment:
I thought the syntax for link is one underscore?
What does two underscore mean?

Reply:
Two underscores makes the reference anonymous I think, making it so if you have duplicate links, the build doesn't complain about duplicates.